src/EditorFeatures/Core.Wpf/Completion/Presentation/RoslynCompletionSet.cs

```diff
@@ -11,6 +11,7 @@
 using Microsoft.VisualStudio.Language.Intellisense;
 using Microsoft.VisualStudio.Text;
 using Microsoft.VisualStudio.Text.Editor;
+using RoslynCompletion = Microsoft.CodeAnalysis.Completion;
 using CompletionItem = Microsoft.CodeAnalysis.Completion.CompletionItem;
```

Comment:
nit: i think (for clarity) purposes, it might make sense to preface all the specific types with Roslyn or VS so that it's always clear in the files which you're referring to.  But if this is a lot of work, i totally get not doing it.